src/components/organisms/BannerAdmin/ConfirmationBannerModal/ConfirmationBannerModal.scss

```diff
@@ -0,0 +1,30 @@
+@import "scss/constants.scss";
+
+.ConfirmationBannerModal {
+  text-align: center;
+
+  &__header {
+    font-size: $font-size--xl;
+    margin: 0;
+  }
+
+  &__message {
+    padding: $spacing--md 0 $spacing--xl;
+  }
+
+  &__buttons {
+    display: flex;
+    justify-content: space-around;
+  }
+}
+
+// Override global styles button
+button.ConfirmationBannerModal__button {
+  width: $button-width;
+  font-weight: $font-weight--500;
+  background-color: $primary--live;
+
+  &--cancel {
+    background-color: opaque-white(0.2);
+  }
+}
```

Comment:
As called out earlier, may be a better pattern for this @sunny-viktoryia @mike-lvov @denisdimitrov ?

Reply:
gave the reference to my workaround here: https://github.com/sparkletown/sparkle/pull/1440#discussion_r658658304

Reply:
fixed 1c2d835